### PR TITLE
Added basic skeleton to home page.

### PR DESCRIPTION
**Pull request by jchen370**
- Added Header, MainContent and Footer Components.
- Added minor styling via LESS file. (I'm aware there is an App.css. We can discuss how we wish to proceed with this.)
- Added usage of FontAwesome.

### CODE REVIEW

File: src/client/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export default class Footer extends React.Component{
```
Comment (RadeshShiwlochan) [inline src/client/components/Footer.js:3]: can we please refrain from using classes? Can we please use functional components instead?

`const Footer = () => {
    return (
     )
 }
`

File: src/client/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export default class Header extends React.Component{
```
Comment (RadeshShiwlochan) [inline src/client/components/Header.js:3]: Please see comments for the Footer.js file

File: src/client/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    state = {
```
Comment (RadeshShiwlochan) [inline src/client/components/Header.js:4]: if we need state, can we use hooks instead? they are much more modern, classes are being phased out

File: src/client/components/MainContent.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { render } from "@testing-library/react";
+
+export default class MainContent extends React.Component {
```
Comment (RadeshShiwlochan) [inline src/client/components/MainContent.js:4]: Please see comment for Footer.js and Header.js files

File: src/client/components/Header.js (hunk 3 above)
Comment (jchen370) [reply src/client/components/Header.js:4]: Sure thing. I'll need to look into hooks. I'm not familiar with them yet.

Comment (RadeshShiwlochan) [reply src/client/components/Header.js:4]: I could be wrong but it doesnt look like you need state for now. I  would just use functional components

**Comment by jchen370** [comment]
Changes have been made. Please review.

**Pull request merged**